src/MediaRatingsKernel.java

```diff
@@ -0,0 +1,77 @@
+import components.standard.Standard;
+
+/**
+ * Media ratings kernel component with primary methods.
+ *
+ * @param <K>
+ *            the type of the key entries
+ * @param <V>
+ *            the type of the value entries
+ */
+public interface MediaRatingsKernel<K, V> extends Standard<MediaRatings<K, V>> {
+    /**
+     * A constant, with value 5, holding the max rating allowed.
+     */
+    int MAXRATING = 5;
```

Comment:
Even here, you specify the rating type as an integer. Maybe remove generic typing and make the pair map String to Integer. Then, you can update you parameter names to be more semantically accurate (e.g., instead of `add(key, value)`, it could be `addRating(mediaName, rating)`)